### PR TITLE
Dict: Add specifi*

### PR DESCRIPTION
**Pull request by sebweb3r**
https://grep.app/search?q=specifig&words=true

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -25994,6 +25995,8 @@ specifiction->specification
 specifictions->specifications
 specificy->specify, specificity, specifically,
 specifid->specified
+specifig->specific
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25998]: Suggest specifying too maybe or do we feel that's too much of a stretch?

**Review by peternewman — COMMENTED** [review]
Just one thought

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:25998]: I looks a bit like a stretch. But I don't mind adding it, if you would like to have it in.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:25998]: I guess it's a bit phonetic too maybe?

Comment (bl-ue) [reply codespell_lib/data/dictionary.txt:25998]: I think it's too much of a stretch too — let's just leave it as-is for right now. wdyt @peternewman?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -25994,6 +25995,8 @@ specifiction->specification
...
+specifigation->specification
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:25999]:
```suggestion
specifigation->specification
specifigations->specifications
```

**Review by bl-ue — APPROVED** [review]

**Pull request merged**